pom.xml

```diff
@@ -43,6 +43,11 @@
 			<scope>runtime</scope>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>de.codecentric</groupId>
```

Comment:
I think it should work with this as well (instead of redefining the dependency again):
`<spring-boot-admin.version>3.1.7</spring-boot-admin.version>`